documentation/appendix_deploying_kubernetes_openshift_cluster.adoc

```diff
@@ -0,0 +1,38 @@
+[appendix]
+== Installing Kubernetes and OpenShift cluster
+
+The easiest way to get started with Kubernetes or OpenShift is using the `Minikube`, `Minishift` or `oc cluster up`
+utilities. This section provides a basic guidance on how to use them. More details are provided on the websites of
+the tools themselves.
+
+=== Kubernetes
+
+In order to interact with a Kubernetes cluster the https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`]
+utility needs to be installed.
+
+The easiest way to get a running Kubernetes cluster is using `Minikube`. `Minikube` can be downloaded and installed
+from https://kubernetes.io/docs/getting-started-guides/minikube/[Kubernetes website]. Once installed, it can be started
+using:
+
+[source]
+minikube start
+
+=== OpenShift
+
+In order to interact with an OpenShift cluster,the https://github.com/openshift/origin/releases[`oc`] utility is needed.
+
+An OpenShift cluster can be started in two different ways. The `oc` utility can start the cluster locally using the
+command:
+
+[source]
+oc cluster up
+
+This command requires Docker to be installed. More information about this way can be found
+https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[here].
+
+Another option is to use `Minishift`. `MiniShift` is an OpenShift installation within a VM. It can be downloaded and
```

Comment:
"...to use Minishift which is ..." ditto as above ?

Reply:
I will keep this in two sentences as discussed above.

Reply:
Same as above, maybe "...which..." can work here but @tombentley is the expert !